xml_inscriptions/bib_only/MA.Bos.MFA.G.84.48.xml

```diff
@@ -35,12 +35,75 @@
-                     <p/>
+                  <msItem class="#text_unknown">
+                     <p>fragmentary inscription mentioning Bacchus and Rhodi(on) or Rhodi(kon)</p>
                   </msItem>
                </msContents>
-               <physDesc><!--objectDesc, handDesc, decoDesc go here--></physDesc>
-               <history><!--summary, origin, provenance go here--></history>
+               <!--  *************************************  <physDesc>  ********************************* -->
+               <physDesc>
+                  
+                  <!--  *************************************  <objectDesc>  *********************************  -->
+                  
+                  <objectDesc ana="#stele">
+                     <supportDesc ana="#stone.marble">
+                        <support>
+                           <p>fragmentary marble stele</p>
+                           <dimensions type="surface" unit="cm">
+                              <height>19.1</height>
+                              <width>53.4</width>
+                              <depth/>
+                           </dimensions>
+                        </support>
+                        <condition ana="#fragment.single">
+                           <p> broken through the center of the inscription</p>
+                        </condition>
+                     </supportDesc>
+                     <!--  *************************************  <layoutDesc>  ********************************* -->
+                     <layoutDesc>
+                        <layout columns="0" writtenLines="10">
+                           <p/>
+                        </layout>
+                     </layoutDesc>
+                  </objectDesc>
+                  
+                  <!--  *************************************  <handDesc>  ********************************* -->
+                  
+                  <handDesc hands="1">
+                     <handNote ana="#inscribed"> letters are small and worn which prevents
+                        legibility (MFA web entry)</handNote>
+                  </handDesc>
+                  
+                  <!--  *************************************  <decoDesc>  *********************************     -->
+                  <decoDesc>
+                     <decoNote>
+                        <p/>
+                     </decoNote>
+                  </decoDesc>
+               </physDesc>
+               
+               <!--  *************************************  <history>  *********************************  -->
+               
+               <history>
+                  
+                  <summary>
+                     <rs/>  <!-- content can be "modern" or "modern-text". A blank value indicates that the inscription is ancient.  -->
+                  </summary>
+                  
+                  <origin>
+                     <date evidence="#palaeography" notBefore="-0900" notAfter="-0151">not later
```

Comment:
This is a massive date range; is it possible to narrow down the `@notBefore` a little more?